app/src/controllers/dashboard/sagas.js

```diff
@@ -162,7 +159,6 @@ function* removeDashboard({ payload: id }) {
       method: 'delete',
     });
     yield put(deleteDashboardSuccessAction(id));
-    yield put({ type: DECREASE_TOTAL_DASHBOARDS_LOCALLY });
```

Comment:
Why did you remove these actions usage while left the validation on 'create new dashboard button'?